.github/workflows/build-lint-test.yml

```diff
@@ -0,0 +1,34 @@
+name: Build, Lint & Test
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+  workflow_dispatch:
+
+concurrency:
+  group: '${{ github.workflow }}-${{ github.head_ref || github.ref }}'
+  cancel-in-progress: true
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+
+    steps:
+    - uses: actions/checkout@v3
+
+    - uses: actions/setup-node@v3
+      with:
+        node-version: '20.8.x'
+
+    - name: build
+      run: npm ci && npm run package
+
+    - name: lint
+      run: npm run lint
```

Comment:
@kayakyakr any idea what I should do to fix this error?

```
Run npm run lint

> contentful-migrations-action@2.3.1 lint
> eslint src/index.ts


/home/runner/work/contentful-action/contentful-action/src/index.ts
  1:1  error  Parsing error: 'import' and 'export' may appear only with 'sourceType: module'

✖ 1 problem (1 error, 0 warnings)

Error: Process completed with exit code 1.
```

Reply:
Install typescript-eslint and reconfigure .eslintrc: https://typescript-eslint.io/getting-started

Reply:
thank you!